src/integration-tests/src/storage.rs

```diff
@@ -38,6 +40,48 @@ pub async fn buckets(builder: storage::client::ClientBuilder) -> Result<()> {
 
     cleanup_stale_buckets(&client, &project_id).await?;
 
+    let bucket_id = random_bucket_id();
+    let bucket_name = format!("projects/_/buckets/{bucket_id}");
+
+    println!("\nTesting create_bucket()");
+    let create = client
+        .create_bucket("projects/_", bucket_id)
+        .set_bucket(
+            Bucket::new()
+                .set_project(format!("projects/{project_id}"))
+                .set_labels([("integration-test", "true")]),
+        )
+        .send()
+        .await?;
+    println!("SUCCESS on create_bucket: {create:?}");
+    assert_eq!(create.name, bucket_name);
+
+    println!("\nTesting get_bucket()");
+    let get = client.get_bucket(&bucket_name).send().await?;
+    println!("SUCCESS on get_bucket: {get:?}");
+    assert_eq!(get.name, bucket_name);
+
+    println!("\nTesting list_buckets()");
+    let mut paginator = client
```

Comment:
nit: I have taken to calling these "items"... they certainly are not a paginator after the `items()` call?

Reply:
Ack, I am going to call them `bucket(s)`. Seems more specific.